app/controllers/notes_controller.rb

```diff
@@ -13,6 +13,7 @@ def create
     @note = @initiative.notes.build(note_params.merge(user_id: current_user.id))
 
     if @note.save
+      UserMailer.team_leader_new_note_reminder(team_member, @initiative, @note).deliver
```

Comment:
Is `deliver_later` an option here? It would be nice to note make the user wait for mail send before the redirect.

Reply:
changed to deliver_later, and verified the new mail has been delivered to team member.